docs/cmd/kn.md

```diff
@@ -16,11 +16,13 @@ Eventing: Manage event subscriptions and channels. Connect up event sources.
       --config string       config file (default is $HOME/.kn/config.yaml)
   -h, --help                help for kn
       --kubeconfig string   kubectl config file (default is $HOME/.kube/config)
+      --plugin-dir string   kn plugin directory (default is value in kn config or $PATH) (default "$PATH")
```

Comment:
Looks weird to have 2 defaults in the description. I'd remove the 2nd sentence.

Reply:
Yup. Good catch